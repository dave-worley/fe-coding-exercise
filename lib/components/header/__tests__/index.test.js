import autoJest from 'autotest/jest';
import autoStorybook from 'autotest/storybook';

import ComponentBase from '../';

export const title = 'Header';
export const Component = ComponentBase;

export const exampleProps = {
  adTargeting: {},
  onToggleLeaderboard() {},
  onLeaderboardChange() {},
};

autoJest(module.exports);
autoStorybook(module.exports);
