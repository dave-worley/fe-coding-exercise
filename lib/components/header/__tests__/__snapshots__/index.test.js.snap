// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header: Default 1`] = `
.glamor-0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
  z-index: 1;
}

.glamor-1 {
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  width: 100%;
  height: 50px;
}

@media (min-width:768px) {
  .glamor-1 {
    height: 70px;
  }
}

<div
  className="glamor-1"
>
  <Styled(header)
    id="site-header"
  >
    <Styled(div)
      className="glamor-0"
    >
      <pure(Component) />
    </Styled(div)>
  </Styled(header)>
</div>
`;
